src/constants/defaultTokenList.json

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
what's the reason to create this file instead of using /token/pancakeswap.json?

Reply:
Great point, I didn't spot that file. I have removed this and updated the import that was using it.